app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__ImageBlockEditor.re

```diff
@@ -37,13 +51,52 @@ let make = (~url, ~caption, ~contentBlock, ~updateContentBlockCB) => {
         <input
           id=captionInputId
           className="mt-1 appearance-none block w-full h-10 bg-white text-gray-800 border rounded py-3 px-3 focus:border-gray-400 leading-tight focus:outline-none focus:bg-white focus:border-gray"
-          onChange={onChange(contentBlock, updateContentBlockCB)}
+          onChange={onChangeCaption(contentBlock, updateContentBlockCB)}
           maxLength=250
           type_="text"
           value=caption
           placeholder="A caption for the image"
         />
       </div>
     </div>
+    <div className="flex border-t justify-end">
+      <div className="flex-1 content-block__action-bar-input p-3">
+        <label htmlFor=widthInputId className="text-sm font-semibold">
+          {React.string("Width")}
+        </label>
+        <select
+          className="cursor-pointer mt-1 appearance-none block w-full bg-white text-gray-800 border rounded py-3 px-3 focus:border-gray-400 leading-tight focus:outline-none focus:bg-white focus:border-gray"
+          id=widthInputId
+          value=widthString
+          onChange={_event => {
+            _event |> ReactEvent.Form.preventDefault;
```

Comment:
Note that the _leading underscore_ in the name of a binding is supposed to signal that it is unused. This also stops the compiler from showing a warning about the unused binding.

This isn't required here since the binding _is_ used.